ObjectCache_Plugin_Admin.php

```diff
@@ -124,4 +127,35 @@ public function w3tc_usage_statistics_summary_from_history( $summary, $history )
 
 		return $summary;
 	}
+
+	/**
+	 * Enqueue disk usage risk acceptance script.
+	 *
+	 * @since X.X.X
+	 *
+	 * @return void
+	 */
+	public function admin_enqueue_scripts() {
+		$page_val = Util_Request::get_string( 'page' );
+		if ( ! empty( $page_val ) && 'w3tc_general' === $page_val ) {
```

Comment:
No need for the `empty()` check -- The utility method always returns a string.
You could use:
```
if ( Util_Request::get_string( 'page' ) === 'w3tc_general' ) {
```

Reply:
Updated